SPECS/xapi.spec

```diff
@@ -520,22 +526,22 @@ done
 %endif
 
 %if %{with dnf_plugin}
-# For xs9, use dnf instead of yum
-echo "%{python3_sitelib}/dnf-plugins/*" >> core-files
-# use dnf instead of yum, clean yum stuff
+# For xs9, use dnf instead of yum, clean yum stuff
 rm -rf %{buildroot}/%{_usr}/lib/yum-plugins/accesstoken.py
 rm -rf %{buildroot}/%{_usr}/lib/yum-plugins/ptoken.py
+rm -rf %{buildroot}/%{_usr}/lib/yum-plugins/xapitoken.py
 rm -rf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/accesstoken.conf
 rm -rf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/ptoken.conf
+rm -rf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/xapitoken.conf
 %else
 # For xs8, use yum
 echo "/etc/yum/pluginconf.d/accesstoken.conf" >> core-files
 echo "/etc/yum/pluginconf.d/ptoken.conf" >> core-files
+echo "/etc/yum/pluginconf.d/xapitoken.conf" >> core-files
```

Comment:
Is this new `xapitoken.conf` without any adverse effects for our XAPI whose RPM management feature we don't use at the moment (we might in the future, when there's an available slot on the roadmap)